instrumentation/cloud.google.com/go/pubsub/README.md

```diff
@@ -17,11 +17,11 @@ $ go get github.com/instana/go-sensor/instrumentation/cloud.google.com/go/pubsub
 Usage
 -----
 
-This module is a drop-in replacement for `cloud.google.com/go/pubsub` in common publisher/subscriber use cases. It provides
+This module is a drop-in replacement for `cloud.google.com/go/pubsub` in a common publisher/subscriber use cases. It provides
```

Comment:
Not sure "a" is needed, since we're talking about plural numbers.

Reply:
Ok, I thought it belongs to the publisher. But I got your point :) I'll revert it